example/gmres/ex_real_A.cpp

```diff
@@ -54,7 +54,7 @@
 
 int main(int argc, char *argv[]) {
 
-  typedef double                            ST;
+  typedef Kokkos::Experimental::half_t      ST;
```

Comment:
Let's change this back to double?  I don't want to push this example with half precision.